jsonnet/utils/set-terminationMessagePolicy.libsonnet

```diff
@@ -0,0 +1,42 @@
+{
+  setTerminationMessagePolicy(o): o {
+    local addTerminationMessagePolicy(o) = o {
+      [if std.setMember(o.kind, ['DaemonSet', 'Deployment', 'ReplicaSet']) then 'spec']+: {
+        template+: {
+          spec+: {
+            containers: [
+              c {
+                terminationMessagePolicy: 'FallbackToLogsOnError',
+              }
+              for c in super.containers
+            ],
+          } + {
+            [if 'initContainers' in super then 'initContainers']: [
+              c {
+                terminationMessagePolicy: 'FallbackToLogsOnError',
+              }
+              for c in super.initContainers
+            ],
+          },
+        },
+      },
+      [if std.setMember(o.kind, ['Prometheus', 'Alertmanager', 'ThanosRuler']) then 'spec']+: {
```

Comment:
I think the set should be ordered (or we can convert the array to a set):

> Sets

> Sets are represented as ordered arrays without duplicates.

> Note that the std.set* functions rely on the uniqueness and ordering on arrays passed to them to work. This can be guaranteed by using std.set(arr). If that is not the case, the functions will quietly return non-meaningful results. 

from https://jsonnet.org/ref/stdlib.html

Reply:
Thanks, that does fix it.